light-client/relayer/src/error.rs

```diff
@@ -0,0 +1,26 @@
+use displaydoc::Display;
+use mc_watcher::error::WatcherDBError;
+
+#[derive(Debug, Display)]
+pub enum Error {
+    /// Thread join error
+    ThreadJoin,
+
+    /// Burned transaction verification error
+    InvalidBurnedTx(mc_light_client_verifier::Error),
```

Comment:
This is not burn specific, to match conventions across the project it should be renamed to `LightClientVerifer(mc_light_client_verifier::Error)` and the docstring should render the error, e.g. `Light client verifeir: {0}`